release/release_gold.sh

```diff
@@ -1,9 +1,23 @@
 #!/bin/bash
 
 source _liferay_common.sh
+file="release-data/release.properties"
```

Comment:
Global variables should contain capital letters.
There should be an empty line between the commands.
The variable name should be more talkative, it should reflect the role (purpose) of the file.
I don't find where this variable is in use.

Reply:
We use it in the "add_tag_to_release" function wit the "lc_get_properties" function.

Reply:
I see it in the get_release_properties_value() only.